diff -urN amavisd-new-20030616.orig/amavisd.conf amavisd-new-20030616/amavisd.conf
--- amavisd-new-20030616.orig/amavisd.conf	Thu Aug 28 12:08:30 2003
+++ amavisd-new-20030616/amavisd.conf	Thu Aug 28 12:10:41 2003
@@ -51,17 +51,20 @@
 # $MYHOME serves as a quick default for some other configuration settings.
 # More refined control is available with each individual setting further down.
 # $MYHOME is not used directly by the program. No trailing slash!
-#$MYHOME = '/var/lib/amavis';   # (default is '/var/amavis')
+$MYHOME = '/var/spool/amavis/runtime';   # (default is '/var/amavis')
 
 # $mydomain serves as a quick default for some other configuration settings.
 # More refined control is available with each individual setting further down.
 # $mydomain is never used directly by the program.
-$mydomain = 'example.com';      # (no useful default)
+# This seems to be prefixed to host's domain name.
+# You may get mail rejects of virus information as send from not 
+# existing host ( virusalert@localhost.whatever.your.domain.is )
+$mydomain = 'localhost';      # (no useful default)
 
 # Set the user and group to which the daemon will change if started as root
 # (otherwise just keep the UID unchanged, and these settings have no effect):
-$daemon_user  = 'vscan';	# (no default;  customary: vscan or amavis)
-$daemon_group = 'sweep';	# (no default;  customary: vscan or amavis)
+$daemon_user  = 'amavis';	# (no default;  customary: vscan or amavis)
+$daemon_group = 'amavis';	# (no default;  customary: vscan or amavis)
 
 # Runtime working directory (cwd), and a place where
 # temporary directories for unpacking mail are created.
@@ -140,7 +143,7 @@
 # and see further down what these two lookup lists really mean.
 #
 # @bypass_virus_checks_acl = qw( . );  # uncomment to DISABLE anti-virus code
-# @bypass_spam_checks_acl  = qw( . );  # uncomment to DISABLE anti-spam code
+@bypass_spam_checks_acl  = qw( . );  # uncomment to DISABLE anti-spam code
 #
 # Any setting can be changed with a new assignment, so make sure
 # you do not unintentionally override these settings further down!
@@ -176,7 +179,7 @@
 # directly, or read from a file, one domain per line; comments and empty lines
 # are ignored, a dot before a domain name implies its subdomains:
 #
-#read_hash(\%local_domains, '/var/amavis/local_domains');
+#read_hash(\%local_domains, '/var/spool/amavis/local_domains');
 
 #or alternatively(B), using a list of regular expressions:
 # $local_domains_re = new_RE( qr'[@.]example\.com$'i );
@@ -271,7 +274,7 @@
 # 3: server, client
 # 4: decompose parts
 # 5: more debug details
-$log_level = 2;		  # (defaults to 0)
+$log_level = 0;		  # (defaults to 0)
 
 # Customizable template for the most interesting log file entry (e.g. with
 # $log_level=0) (take care to properly quote Perl special characters like '\')
@@ -310,12 +313,12 @@
 # or later; second argument is ignored otherwise. See PerlIO::encoding,
 # Encode::PerlIO and perluniintro man pages.
 #
-# $notify_sender_templ      = read_text('/var/amavis/notify_sender.txt');
-# $notify_virus_sender_templ= read_text('/var/amavis/notify_virus_sender.txt');
-# $notify_virus_admin_templ = read_text('/var/amavis/notify_virus_admin.txt');
-# $notify_virus_recips_templ= read_text('/var/amavis/notify_virus_recips.txt');
-# $notify_spam_sender_templ = read_text('/var/amavis/notify_spam_sender.txt');
-# $notify_spam_admin_templ  = read_text('/var/amavis/notify_spam_admin.txt');
+# $notify_sender_templ      = read_text('/var/spool/amavis/notify_sender.txt');
+# $notify_virus_sender_templ= read_text('/var/spool/amavis/notify_virus_sender.txt');
+# $notify_virus_admin_templ = read_text('/var/spool/amavis/notify_virus_admin.txt');
+# $notify_virus_recips_templ= read_text('/var/spool/amavis/notify_virus_recips.txt');
+# $notify_spam_sender_templ = read_text('/var/spool/amavis/notify_spam_sender.txt');
+# $notify_spam_admin_templ  = read_text('/var/spool/amavis/notify_spam_admin.txt');
 
 # If notification template files are collectively available in some directory,
 # use read_l10n_templates which calls read_text for each known template.
@@ -402,25 +405,25 @@
 # Bounces or rejects produce non-delivery status notification anyway.
 
 # Notify virus sender?
-#$warnvirussender = 1;	# (defaults to false (undef))
+$warnvirussender = 1;	# (defaults to false (undef))
 
 # Notify spam sender?
-#$warnspamsender = 1;	# (defaults to false (undef))
+$warnspamsender = 1;	# (defaults to false (undef))
 
 # Notify sender of banned files?
-#$warnbannedsender = 1;	# (defaults to false (undef))
+$warnbannedsender = 1;	# (defaults to false (undef))
 
 # Notify sender of syntactically invalid header containing non-ASCII characters?
 #$warnbadhsender = 1;	# (defaults to false (undef))
 
 # Notify virus (or banned files) RECIPIENT?
 #  (not very useful, but some policies demand it)
-#$warnvirusrecip = 1;	# (defaults to false (undef))
-#$warnbannedrecip = 1;	# (defaults to false (undef))
+$warnvirusrecip = 1;	# (defaults to false (undef))
+$warnbannedrecip = 1;	# (defaults to false (undef))
 
 # Notify also non-local virus/banned recipients if $warn*recip is true?
 #  (including those not matching local_domains*)
-#$warn_offsite = 1;	# (defaults to false (undef), i.e. only notify locals)
+$warn_offsite = 1;	# (defaults to false (undef), i.e. only notify locals)
 
 
 # Treat envelope sender address as unreliable and don't send sender
@@ -498,7 +501,7 @@
 #   or a directory (no trailing slash)
 #   (the default value is undef, meaning no quarantine)
 #
-$QUARANTINEDIR = '/var/virusmails';
+$QUARANTINEDIR = '/var/spool/amavis/virusmails';
 
 #$virus_quarantine_method = "local:virus-%i-%n";    # default
 #$spam_quarantine_method  = "local:spam-%b-%i-%n";  # default
@@ -911,7 +914,7 @@
 # a hash lookup table can be read from a file,
 # one address per line, comments and empty lines are permitted:
 #
-# read_hash(\%whitelist_sender, '/var/amavis/whitelist_sender');
+# read_hash(\%whitelist_sender, '/var/spool/amavis/whitelist_sender');
 
 # ... or set directly:
 
@@ -1152,12 +1155,12 @@
 # ['Sophos SAVI', \&sophos_savi ],
 
 # ### http://clamav.elektrapro.com/
-# ['Clam Antivirus-clamd',
-#   \&ask_daemon, ["CONTSCAN {}\n", '/var/amavis/clamd'],
-#   qr/\bOK$/, qr/\bFOUND$/,
-#   qr/^.*?: (?!Infected Archive)(.*) FOUND$/ ],
-# # NOTE: run clamd under the same user as amavisd,
-# # match the socket name in clamav.conf to the socket name in this entry
+
+  ['Clam Antivirus-clamd',
+   \&ask_daemon, ["CONTSCAN {}\n", '/var/lib/clamav/clamd.socket'],
+   qr/\bOK$/, qr/\bFOUND$/,
+   qr/^.*?: (?!Infected Archive)(.*) FOUND$/ ],
+# NOTE: match the socket name in clamav.conf to the socket name in this entry
 
 # ### http://www.openantivirus.org/
 # ['OpenAntiVirus ScannerDaemon (OAV)',
@@ -1241,13 +1244,18 @@
     '-sec -nex {}', [0], [100],
     qr/was infected by virus (.+)/ ],
 
-  ['MkS_Vir for Linux (beta)', ['mks32','mks'],
-    '-s {}/*', [0], [1,2],
-    qr/--[ \t]*(.+)/ ], 
-
-  ['MkS_Vir daemon',
-    'mksscan', '-s -q {}', [0], [1..7],
-    qr/^... (\S+)/ ],
+  #['MkS_Vir daemon',
+  #  'mksscan', '-s -q {}', [0], [1..7],
+  #  qr/^... (\S+)/ ],
+
+  ['MkS_Vir Daemon for Linux', 
+   'mkschk','-s {}/*', [0], [1,2],
+   qr/--[ \t]*(.+)/ ],
+
+  # Moved to secondary
+  #['MkS_Vir for Linux (beta)', ['mks32','mks'],
+  #  '-s {}/*', [0], [1,2],
+  #  qr/--[ \t]*(.+)/ ], 
 
   ### http://www.nod32.com/
   ['ESET Software NOD32', 'nod32',
@@ -1346,6 +1354,10 @@
   ['Trend Micro FileScanner', ['/etc/iscan/vscan','vscan'],
     '-a {}', [0], qr/Found virus/, qr/Found virus (.+) in/ ],
 
+  ['MkS_Vir for Linux (beta)', ['mks32','mks'],
+    '-s {}/*', [0], [1,2],
+    qr/--[ \t]*(.+)/ ], 
+
 # Commented out because the name 'sweep' clashes with the Debian package of
 # the same name. Make sure the correct sweep is found in the path when enabling
 #
