diff -dur amavisd-new-20040701.orig/helper-progs/configure.in amavisd-new-20040701/helper-progs/configure.in
--- amavisd-new-20040701.orig/helper-progs/configure.in	2004-04-11 01:51:52.000000000 +0200
+++ amavisd-new-20040701/helper-progs/configure.in	2004-08-02 11:07:56.608560250 +0200
@@ -138,14 +138,14 @@
 
 dnl find libmilter.a and libsm.a
 SMPATH=`echo ../sendmail-*/obj.*/libmilter`
-AC_PATH_PROG(LIBMILTER, libmilter.a, no, $MILTERLIB:$SMPATH:/usr/local/lib:/lib:/usr/lib:/usr/lib/libmilter)
+AC_PATH_PROG(LIBMILTER, libmilter.a, no, $MILTERLIB:$SMPATH:/usr/local/lib64:/lib64:/usr/lib64:/usr/lib64/libmilter)
 SMPATH=`echo ../sendmail-*/obj.*/libsm`
-AC_PATH_PROG(LIBSM, libsm.a, no, $SMPATH:/usr/local/lib:/lib:/usr/lib:/usr/lib/libmilter)
+AC_PATH_PROG(LIBSM, libsm.a, no, $SMPATH:/usr/local/lib64:/lib64:/usr/lib64:/usr/lib64/libmilter)
 if test "$LIBSM" = "no" ; then
 	# try libsmutil
 	unset ac_cv_path_LIBSM
 	SMPATH=`echo ../sendmail-*/obj.*/libsmutil`
-	AC_PATH_PROG(LIBSM, libsmutil.a, no, $SMPATH:/usr/local/lib:/lib:/usr/lib:/usr/lib/libmilter)
+	AC_PATH_PROG(LIBSM, libsmutil.a, no, $SMPATH:/usr/local/lib64:/lib64:/usr/lib64:/usr/lib64/libmilter)
 fi
 
 dnl find Sendmail
Only in amavisd-new-20040701/helper-progs: configure.in~
