diff -Nur amavisd-new-20030616.orig/helper-progs/amavis-milter-based-on-1.1.2.3.2.40-v2.c amavisd-new-20030616/helper-progs/amavis-milter-based-on-1.1.2.3.2.40-v2.c
--- amavisd-new-20030616.orig/helper-progs/amavis-milter-based-on-1.1.2.3.2.40-v2.c	2004-04-01 02:42:23.000000000 +0200
+++ amavisd-new-20030616/helper-progs/amavis-milter-based-on-1.1.2.3.2.40-v2.c	2004-05-06 16:29:22.000000000 +0200
@@ -884,7 +884,7 @@
     srand48(time(NULL) ^ (mypid + (mypid << 15)));
 #endif
 
-    umask(0007);
+    umask(0077);
 
     /* Process command line options */
     while ((c = getopt(argc, argv, args)) != -1) {
