diff -ur amavisd-new-20020517.orig/amavis/amavisd.in amavisd-new-20020517/amavis/amavisd.in
--- amavisd-new-20020517.orig/amavis/amavisd.in	Tue Sep 17 15:27:37 2002
+++ amavisd-new-20020517/amavis/amavisd.in	Tue Sep 17 15:42:09 2002
@@ -310,6 +310,8 @@
 # (e.g. for Postfix add 'recipient_delimiter = +' to main.cf)
 
 use vars '$recipient_delimiter';  $recipient_delimiter = '+';
+use vars '$replace_existing_extension';
+$replace_existing_extension = 1;   # false: append ext, true: replace ext
 
 
 # Spam in general
@@ -1027,6 +1029,10 @@
 		    next  if !lookup($mailbox, undef, \@local_domains);
 		    my($localpart,$domain) = ($mailbox =~ /^(.*)(\@[^@]*)$/) ?
 						($1,$2) : ($mailbox,'');
+		    if ($replace_existing_extension) {
+		    	# strip away existing address extensions
+			$localpart =~ s/^(.*?)\Q$recipient_delimiter\E.*$/$1/;
+		    }
 		    $mailbox = $localpart . $recipient_delimiter
 				. (@virusname ? $addr_extension_virus 
 				  : ($spam_found ? $addr_extension_spam : ''))
diff -ur amavisd-new-20020517.orig/amavis/amavisd.conf.in amavisd-new-20020517/amavis/amavisd.conf.in
--- amavisd-new-20020517.orig/amavis/amavisd.conf.in	Tue Sep 17 15:09:47 2002
+++ amavisd-new-20020517/amavis/amavisd.conf.in	Tue Sep 17 15:43:31 2002
@@ -489,6 +489,7 @@
 # (e.g. for Postfix add 'recipient_delimiter = +' to main.cf)
 
 $recipient_delimiter = '+';
+$replace_existing_extension = 1; # false: append ext, true: replace ext
 
 
 # SMTP sender whitelist ACL to bypass spam checking: %whitelist_sender, @whitelist_sender
