diff -urp amavis-v2.12.1.orig/amavisd-agent amavis-v2.12.1/amavisd-agent
--- amavis-v2.12.1.orig/amavisd-agent	2020-11-13 13:55:37.000000000 +0100
+++ amavis-v2.12.1/amavisd-agent	2021-01-24 21:28:13.409592502 +0100
@@ -53,7 +53,7 @@ use BerkeleyDB;
 
 my($dbfile) = 'snmp.db';
 my($db_home) =  # DB databases directory
-  defined $ENV{'AMAVISD_DB_HOME'} ? $ENV{'AMAVISD_DB_HOME'} : '/var/amavis/db';
+  defined $ENV{'AMAVISD_DB_HOME'} ? $ENV{'AMAVISD_DB_HOME'} : '/var/spool/amavis/db';
 
 my($wakeuptime) = 10;  # -w, sleep time in seconds, may be fractional
 my($repeatcount);      # -c, repeat count (when defined)
diff -urp amavis-v2.12.1.orig/amavisd-nanny amavis-v2.12.1/amavisd-nanny
--- amavis-v2.12.1.orig/amavisd-nanny	2020-11-13 13:55:37.000000000 +0100
+++ amavis-v2.12.1/amavisd-nanny	2021-01-24 21:28:13.409592502 +0100
@@ -61,7 +61,7 @@ my($activettl) = 10*60; # stuck active c
 
 my($dbfile) = 'nanny.db';
 my($db_home) =  # DB databases directory
-  defined $ENV{'AMAVISD_DB_HOME'} ? $ENV{'AMAVISD_DB_HOME'} : '/var/amavis/db';
+  defined $ENV{'AMAVISD_DB_HOME'} ? $ENV{'AMAVISD_DB_HOME'} : '/var/spool/amavis/db';
 my($wakeuptime) = 2;  # -w, sleep time in seconds, may be fractional
 my($repeatcount);     # -c, repeat count (when defined)
 
