diff -ur amavisd-new-20020517.orig/amavis/amavisd.conf.in amavisd-new-20020517/amavis/amavisd.conf.in
--- amavisd-new-20020517.orig/amavis/amavisd.conf.in	Fri Sep  6 19:36:07 2002
+++ amavisd-new-20020517/amavis/amavisd.conf.in	Fri Sep  6 19:38:02 2002
@@ -194,7 +194,7 @@
 # Location to put infected mail into
 # - empty for not quarantining, may be a file (mailbox),
 #   or a directory (no trailing slash)
-$QUARANTINEDIR = "/var/virusmails";
+$QUARANTINEDIR = "/var/spool/amavis/virusmails";
 
 # A finer control of quarantining is available through
 # variable $virus_quarantine_to as follows:
diff -ur amavisd-new-20020517.orig/amavis/amavisd.in.all amavisd-new-20020517/amavis/amavisd.in.all
--- amavisd-new-20020517.orig/amavis/amavisd.in.all	Fri May 17 22:53:19 2002
+++ amavisd-new-20020517/amavis/amavisd.in.all	Fri Sep  6 19:36:50 2002
@@ -3477,7 +3477,7 @@
 $spamassasin_obj = Mail::SpamAssassin->new({
 	dont_copy_prefs => 1, local_tests_only => 1 });
 $spamassasin_obj->compile_now;	# ensure all modules etc. are preloaded
-%razor_options = ( listfile  => '/var/amavis/razor.lst' );
+%razor_options = ( listfile  => '/var/spool/amavis/razor.lst' );
 
 # Read config file, which may override default settings
 read_config();
@@ -3506,7 +3506,7 @@
   'spam-quarantine'  => sub { ($QUARANTINEDIR, "$VIRUSFILE.gz") },
   'archive-by-sender' =>		# just an example
 	sub { my($s) = $SENDER; $s =~ s/[^a-zA-Z0-9._=@]/-/; $s =~ s/\@/=/;
-	      ( '/var/virusmails',	# directory (qmail style mailbox)
+	      ( '/var/spool/amavis/virusmails',	# directory (qmail style mailbox)
 		sprintf("archive-%s-%s-%05d.gz",  # suggested file name
 			$s, strftime("%Y%m%d-%H%M%S",localtime), $$) );
 	    },
