diff -uNr amavisd-new-20030616/amavisd.conf-sample amavisd-new-20030616~/amavisd.conf-sample
--- amavisd-new-20030616/amavisd.conf-sample	Tue Mar  9 03:21:38 2004
+++ amavisd-new-20030616~/amavisd.conf-sample	Sun Mar 14 13:37:30 2004
@@ -51,27 +51,31 @@
 # $MYHOME serves as a quick default for some other configuration settings.
 # More refined control is available with each individual setting further down.
 # $MYHOME is not used directly by the program. No trailing slash!
-#$MYHOME = '/var/lib/amavis';   # (default is '/var/amavis')
+$MYHOME = '/var/spool/amavis';   # (default is '/var/amavis')
 
 # $mydomain serves as a quick default for some other configuration settings.
 # More refined control is available with each individual setting further down.
 # $mydomain is never used directly by the program.
-$mydomain = 'example.com';      # (no useful default)
+# $mydomain = 'example.com';      # (no useful default)
+
+# This seems to be prefixed to host's domain name.
+# You may get mail rejects of virus information as send from not
+# existing host ( virusalert@localhost.whatever.your.domain.is )
+$mydomain = 'mail.whatever.your.domain.is';      # (no useful default)
 
 # $myhostname = 'host.example.com';  # fqdn of this host, default by uname(3)
 
 # Set the user and group to which the daemon will change if started as root
 # (otherwise just keeps the UID unchanged, and these settings have no effect):
-$daemon_user  = 'vscan';	# (no default;  customary: vscan or amavis)
-$daemon_group = 'sweep';	# (no default;  customary: vscan or amavis)
+$daemon_user  = 'amavis';	# (no default;  customary: vscan or amavis)
+$daemon_group = 'amavis';	# (no default;  customary: vscan or amavis)
 
 # Runtime working directory (cwd), and a place where
 # temporary directories for unpacking mail are created.
 # (no trailing slash, may be a scratch file system)
-$TEMPBASE = $MYHOME;	        # (must be set if other config vars use is)
-#$TEMPBASE = "$MYHOME/tmp";     # prefer to keep home dir /var/amavis clean?
+$TEMPBASE = "$MYHOME/runtime";	        # (must be set if other config vars use is)
 
-#$db_home = "$MYHOME/db";	# DB databases directory, default "$MYHOME/db"
+$db_home = "/var/spool/amavis/db";	# DB databases directory, default "$MYHOME/db"
 
 # $helpers_home sets environment variable HOME, and is passed as option
 # 'home_dir_for_helpers' to Mail::SpamAssassin::new. It should be a directory
@@ -106,6 +106,8 @@
 # as an IP address or a DNS name (A or CNAME, but MX is ignored)
 #$forward_method = 'smtp:[127.0.0.1]:10025';  # where to forward checked mail
 #$notify_method = $forward_method;            # where to submit notifications
+
+# @auth_mech_avail = ('PLAIN','LOGIN'); # empty list disables incoming AUTH
 
 # To make it possible for several hosts to share one content checking daemon,
 # the IP address and/or the port number in $forward_method and $notify_method
@@ -143,7 +148,8 @@
 # and see further down what these two lookup lists really mean.
 #
 # @bypass_virus_checks_maps = (1);  # uncomment to DISABLE anti-virus code
-# @bypass_spam_checks_maps  = (1);  # uncomment to DISABLE anti-spam code
+@bypass_spam_checks_maps  = (1);  # uncomment to DISABLE anti-spam code
+
 #
 # Any setting can be changed with a new assignment, so make sure
 # you do not unintentionally override these settings further down!
@@ -267,7 +273,7 @@
 #$SYSLOG_LEVEL = 'user.info';     # (facility.priority, default 'mail.info')
 
 # Log file (if not using syslog)
-$LOGFILE = "$MYHOME/amavis.log";  # (defaults to empty, no log)
+# $LOGFILE = "$MYHOME/amavis.log";  # (defaults to empty, no log)
 
 #NOTE: levels are not strictly observed and are somewhat arbitrary
 # 0: startup/exit/failure messages, viruses detected
@@ -276,7 +282,7 @@
 # 3: server, client
 # 4: decompose parts
 # 5: more debug details
-$log_level = 2;		  # (defaults to 0)
+# $log_level = 2;		  # (defaults to 0)
 
 # Customizable template for the most interesting log file entry (e.g. with
 # $log_level=0) (take care to properly quote Perl special characters like '\')
@@ -408,22 +414,22 @@
 # Bounces or rejects produce non-delivery status notification anyway.
 
 # Notify virus sender?
-#$warnvirussender = 1;	# (defaults to false (undef))
+$warnvirussender = 1;	# (defaults to false (undef))
 
 # Notify spam sender?
-#$warnspamsender = 1;	# (defaults to false (undef))
+$warnspamsender = 1;	# (defaults to false (undef))
 
 # Notify sender of banned files?
-#$warnbannedsender = 1;	# (defaults to false (undef))
+$warnbannedsender = 1;	# (defaults to false (undef))
 
 # Notify sender of syntactically invalid header containing non-ASCII characters?
-#$warnbadhsender = 1;	# (defaults to false (undef))
+# $warnbadhsender = 1;	# (defaults to false (undef))
 
 # Notify virus (or banned files or bad headers) RECIPIENT?
 #  (not very useful, but some policies demand it)
-#$warnvirusrecip = 1;	# (defaults to false (undef))
-#$warnbannedrecip = 1;	# (defaults to false (undef))
-#$warnbadhrecip = 1;	# (defaults to false (undef))
+$warnvirusrecip = 1;	# (defaults to false (undef))
+$warnbannedrecip = 1;	# (defaults to false (undef))
+$warnbadhrecip = 1;	# (defaults to false (undef))
 
 # Notify also non-local virus/banned recipients if $warn*recip is true?
 #  (including those not matching local_domains*)
@@ -516,7 +522,8 @@
 #   or a directory (no trailing slash)
 #   (the default value is undef, meaning no quarantine)
 #
-$QUARANTINEDIR = '/var/virusmails';
+# $QUARANTINEDIR = '/var/virusmails';
+$QUARANTINEDIR = '/var/spool/amavis/virusmails';
 
 #$virus_quarantine_method = "local:virus-%i-%n";    # default
 #$spam_quarantine_method  = "local:spam-%b-%i-%n";  # default
@@ -1193,13 +1200,12 @@
 # ['Sophos SAVI', \&sophos_savi ],
 
 # ### http://www.clamav.net/
-# ['ClamAV-clamd',
-#   \&ask_daemon, ["CONTSCAN {}\n", "/var/run/clamav/clamd"],
-#   qr/\bOK$/, qr/\bFOUND$/,
-#   qr/^.*?: (?!Infected Archive)(.*) FOUND$/ ],
-# # NOTE: run clamd under the same user as amavisd;  match the socket
-# # name (LocalSocket) in clamav.conf to the socket name in this entry
-# # When running chrooted one may prefer: ["CONTSCAN {}\n","$MYHOME/clamd"],
+  ['Clam Antivirus-clamd',
+    \&ask_daemon, ["CONTSCAN {}\n", '/var/lib/clamav/clamd.socket'],
+    qr/\bOK$/, qr/\bFOUND$/,
+    qr/^.*?: (?!Infected Archive)(.*) FOUND$/ ],
+
+# NOTE: match the socket name in clamav.conf to the socket name in this entry
 
 # ### http://www.openantivirus.org/
 # ['OpenAntiVirus ScannerDaemon (OAV)',
@@ -1316,9 +1322,10 @@
     '-sec -nex {}', [0], [100],
     qr/was infected by virus (.+)/ ],
 
-  ['MkS_Vir for Linux (beta)', ['mks32','mks'],
-    '-s {}/*', [0], [1,2],
-    qr/--[ \t]*(.+)/ ], 
+#  Moved to secondary scanner
+#  ['MkS_Vir for Linux (beta)', ['mks32','mks'],
+#    '-s {}/*', [0], [1,2],
+#    qr/--[ \t]*(.+)/ ], 
 
   ['MkS_Vir daemon',
     'mksscan', '-s -q {}', [0], [1..7],
@@ -1444,6 +1451,10 @@
 # ],
 # # other options to consider: -mime -oe -idedir=/usr/local/sav
 
+   ['MkS_Vir for Linux (beta)', ['mks32','mks'],
+     '-s {}/*', [0], [1,2],
+     qr/--[ \t]*(.+)/ ],
+
 # always succeeds (uncomment to consider mail clean if all other scanners fail)
 # ['always-clean', sub {0}],
 
