--- ./amavisd.org	Thu Mar 11 12:09:45 2004
+++ ./amavisd	Thu Mar 11 12:10:33 2004
@@ -5928,7 +5928,7 @@
 #   my($notify_virus_admin_only_if_sender_is_local) = 0;
 
     # try to find a per-sender administrator
-    my($admin) = lookup($msginfo->sender, \%virus_admin,$virus_admin,$mailto);
+    my($admin) = lookup($msginfo->sender_source, \%virus_admin,$virus_admin,$mailto);
     if ($admin eq '') {
 	do_log(4, "Skip virus_admin notification for <".$msginfo->sender.
 		  ">, no admin specified");
