--- amavisd-new-2.1.2/amavisd-agent	2004-09-06 17:56:26.000000000 +0000
+++ amavisd-new-2.1.2.new/amavisd-agent	2004-09-07 21:09:49.980116920 +0000
@@ -107,7 +107,7 @@
 # main program starts here
   $SIG{INT} = sub { die "\n" };  # do the END code block
   my($env) = BerkeleyDB::Env->new(
-    '-Home'=>'/var/amavis/db', '-Flags'=> DB_INIT_CDB | DB_INIT_MPOOL);
+    '-Home'=>'/var/spool/amavis/db', '-Flags'=> DB_INIT_CDB | DB_INIT_MPOOL);
   defined $env or die "BDB no env: $BerkeleyDB::Error $!";
   my($db) = BerkeleyDB::Hash->new(
     '-Filename'=>'snmp.db', '-Flags'=>DB_RDONLY, '-Env'=>$env );
--- amavisd-new-2.1.2/amavisd-nanny	2004-09-06 17:56:31.000000000 +0000
+++ amavisd-new-2.1.2.new/amavisd-nanny	2004-09-07 21:10:29.346132376 +0000
@@ -51,7 +51,7 @@
 my($activettl) = 10*60; # stuck active children are sent a SIGTERM after this
                         # many seconds
 
-my($db_home) = '/var/amavis/db';  # DB databases directory
+my($db_home) = '/var/spool/amavis/db';  # DB databases directory
 my($dbfile)  = 'nanny.db';
 my($wakeuptime) = 2;  # seconds
 
