diff -urN amavisd-new-20020630.org/amavis/amavisd.conf.in amavisd-new-20020630/amavis/amavisd.conf.in
--- amavisd-new-20020630.org/amavis/amavisd.conf.in	Mon Sep  9 21:11:20 2002
+++ amavisd-new-20020630/amavis/amavisd.conf.in	Mon Sep  9 21:11:31 2002
@@ -39,6 +39,9 @@
 $avp = "@avp@";
 $AVPDIR = dirname($avp);
 
+# Clam AntiVirus http://www.konarski.edu.pl/~zolw/clam.html
+$clamav = "@clamav@";
+
 # KasperskyLab AVPDaemon / AvpDaemonClient
 #
 # use AvpDaemon and AvpDaemonClient
diff -urN amavisd-new-20020630.org/amavis/amavisd.in amavisd-new-20020630/amavis/amavisd.in
--- amavisd-new-20020630.org/amavis/amavisd.in	Mon Sep  9 21:11:20 2002
+++ amavisd-new-20020630/amavis/amavisd.in	Mon Sep  9 21:12:05 2002
@@ -76,7 +76,7 @@
 	    $virus_quarantine_to $spam_quarantine_to
 	)],
 	'av' => [qw(
-	    $antivir $avp $avpdc $AVPDIR $csav $drweb $fprot $fprotd $fsav
+	    $antivir $avp $avpdc $AVPDIR $csav $drweb $fprot $fprotd $fsav $clamav
 	    $inocucmd $mks $nod32 $nod32cli $norman $oav $panda $rav $sophos
 	    $sophos_ide_path $cscmdline $scs_host $scs_port $uvscan $vbengcl
 	    $vexira $vfind $vscan $sophie_sockname $trophie_sockname
@@ -3715,6 +3715,7 @@
 @avp_scanner_frag@
 @avpdc_scanner_frag@
 @cai_scanner_frag@
+@clamav_scanner_frag@
 @csav_scanner_frag@
 @cyber_scanner_frag@
 @drweb_scanner_frag@
diff -urN amavisd-new-20020630.org/amavis/av/clamav amavisd-new-20020630/amavis/av/clamav
--- amavisd-new-20020630.org/amavis/av/clamav	Thu Jan  1 01:00:00 1970
+++ amavisd-new-20020630/amavis/av/clamav	Mon Sep  9 21:11:31 2002
@@ -0,0 +1,21 @@
+#
+# clamav (GPL) http://www.konarski.edu.pl/~zolw/clam.html
+#
+
+
+if ($clamav) {
+	do_log(2, "Using $clamav");
+        chop($output = `$clamav --stdout -r -w --one-virus $TEMPDIR/parts 2>&1`);
+        $errval = retcode($?);
+	$scanner_errors &= $errval;
+        do_log(2,$output);
+        if ($errval != 0) {
+                if ($errval >= 51 || $errval <= 53) {
+                        @virusname = ($output =~ /.*: (.+) FOUND/g);
+                        do_virus($output);
+                } else {
+                        do_log(0,"Virus scanner failure: $clamav (error code: $errval)");
+                }
+        }
+}
+
diff -urN amavisd-new-20020630.org/configure.in amavisd-new-20020630/configure.in
--- amavisd-new-20020630.org/configure.in	Mon Sep  9 21:11:20 2002
+++ amavisd-new-20020630/configure.in	Mon Sep  9 21:15:39 2002
@@ -397,6 +397,7 @@
   avp_scanner_frag=$srcdir/amavis/av/avp
   avpdc_scanner_frag=$srcdir/amavis/av/avpdc
   cai_scanner_frag=$srcdir/amavis/av/cai
+  clamav_scanner_frag=$srcdir/amavis/av/clamav
   csav_scanner_frag=$srcdir/amavis/av/csav
   cyber_scanner_frag=$srcdir/amavis/av/cyber
   drweb_scanner_frag=$srcdir/amavis/av/drweb
@@ -472,6 +473,17 @@
   cai_scanner_frag=$srcdir/amavis/av/cai
 fi
 
+### ClamAntiVirus http://www.konarski.edu.pl/~zolw/clam.html
+clamav_scanner_frag=/dev/null
+AC_CHECKING([*** Clam AntiVirus Scanner ***])
+AC_PATH_PROG(clamscan, clamscan, , $PATH:/bin:/usr/bin:/usr/local/bin)
+$clamav = $clamscan
+AC_SUBST(clamav)
+if test "x$clamscan" != "x" ; then
+  avscanner="$avscanner Clam AntiVirus Scanner"
+  clamav_scanner_frag=$srcdir/amavis/av/clamav
+fi
+
 ### Command AntiVirus for Linux
 csav_scanner_frag=/dev/null
 AC_CHECKING([*** Command AntiVirus for Linux ***])
@@ -740,7 +752,7 @@
 fi
 
 AC_MSG_CHECKING([if any virus scanners are installed at all])
-if test "x$antivir$uvscan$ufindvir$fvsco$sophos$avp$avpdc$fsav$vscan$vfind$inocucmd$rav$nod32$nod32cli$csav$vbengcl$sophie$trophie$panda$fprot$fprotd$oav$cscmdline$drweb$mks$vexira$norman" = "x" ; then
+if test "x$clamav$antivir$uvscan$ufindvir$fvsco$sophos$avp$avpdc$fsav$vscan$vfind$inocucmd$rav$nod32$nod32cli$csav$vbengcl$sophie$trophie$panda$fprot$fprotd$oav$cscmdline$drweb$mks$vexira$norman" = "x" ; then
   AC_MSG_RESULT(no)
   AC_MSG_ERROR([
 
@@ -887,6 +899,7 @@
 AC_SUBST_FILE(avp_scanner_frag)
 AC_SUBST_FILE(avpdc_scanner_frag)
 AC_SUBST_FILE(cai_scanner_frag)
+AC_SUBST_FILE(clamav_scanner_frag)
 AC_SUBST_FILE(csav_scanner_frag)
 AC_SUBST_FILE(cyber_scanner_frag)
 AC_SUBST_FILE(drweb_scanner_frag)
