diff -ur amavisd-new-20021227.orig/amavisd amavisd-new-20021227/amavisd
--- amavisd-new-20021227.orig/amavisd	Mon Dec 30 17:36:10 2002
+++ amavisd-new-20021227/amavisd	Tue Jan  7 18:00:57 2003
@@ -214,7 +214,7 @@
 #
 # Receiving mail related
 
-# $unix_socketname = '/var/amavis/amavisd.sock'; # traditional amavis client protocol
+# $unix_socketname = '/var/spool/amavis/runtime/amavisd.sock'; # traditional amavis client protocol
 # $inet_socket_port = 10024;      # accept SMTP on this TCP port
 $inet_socket_bind = '127.0.0.1';  # limit socket bind to loopback interface
 
@@ -223,7 +223,7 @@
 $gets_addr_in_quoted_form = 0;
 
 # Temporary directory
-$TEMPBASE = "/var/amavis";
+$TEMPBASE = "/var/spool/amavis/runtime";
 
 $notify_method  = 'smtp:127.0.0.1:10025';
 $forward_method = 'smtp:127.0.0.1:10025';
diff -ur amavisd-new-20021227.orig/amavisd.conf amavisd-new-20021227/amavisd.conf
--- amavisd-new-20021227.orig/amavisd.conf	Fri Dec 27 16:04:17 2002
+++ amavisd-new-20021227/amavisd.conf	Tue Jan  7 18:00:02 2003
@@ -40,7 +40,7 @@
 $daemon_group = 'amavis';	# (no default)
 
 # Runtime directory (no trailing slash, defaults to '/var/amavis')
-$TEMPBASE = '/var/amavis';
+$TEMPBASE = '/var/spool/amavis/runtime';
 
 
 # MTA SETTINGS, UNCOMMENT AS APPROPRIATE, defaults to 'smtp:127.0.0.1:10025'
@@ -59,7 +59,7 @@
 #$notify_method = 'pipe:flags=q argv=/usr/sbin/sendmail -i -f ${sender} -- ${recipient}';
 
 
-#$daemon_chroot_dir = '/var/amavis';    # (default is undef => do not chroot)
+#$daemon_chroot_dir = '/var/spool/amavis';    # (default is undef => do not chroot)
 
 #$pid_file  = "$TEMPBASE/amavisd.pid";  # (default: "$TEMPBASE/amavisd.pid")
 #$lock_file = "$TEMPBASE/amavisd.lock"; # (default: "$TEMPBASE/amavisd.lock")
@@ -221,12 +221,12 @@
 # assigning new text to template variables, or by reading template text
 # from files:
 #
-# $notify_sender_templ      = read_text('/var/amavis/notify_sender.txt');
-# $notify_virus_sender_templ= read_text('/var/amavis/notify_virus_sender.txt');
-# $notify_virus_admin_templ = read_text('/var/amavis/notify_virus_admin.txt');
-# $notify_virus_recips_templ= read_text('/var/amavis/notify_virus_recips.txt');
-# $notify_spam_sender_templ = read_text('/var/amavis/notify_spam_sender.txt');
-# $notify_spam_admin_templ  = read_text('/var/amavis/notify_spam_admin.txt');
+# $notify_sender_templ      = read_text('/var/spool/amavis/notify_sender.txt');
+# $notify_virus_sender_templ= read_text('/var/spool/amavis/notify_virus_sender.txt');
+# $notify_virus_admin_templ = read_text('/var/spool/amavis/notify_virus_admin.txt');
+# $notify_virus_recips_templ= read_text('/var/spool/amavis/notify_virus_recips.txt');
+# $notify_spam_sender_templ = read_text('/var/spool/amavis/notify_spam_sender.txt');
+# $notify_spam_admin_templ  = read_text('/var/spool/amavis/notify_spam_admin.txt');
 
 # Treat envelope sender address as unreliable and don't send 
 # sender notification if name(s) of detected virus(es) match the list.
@@ -255,9 +255,9 @@
 #   for compatibility, which in turn defaults to undef (empty) )
 #
 $mailfrom_notify_sender    = '';   # null reserse path. Should not be changed!
-$mailfrom_notify_admin     = 'virusalert@example.com';
-$mailfrom_notify_recip     = 'virusalert@example.com';
-$mailfrom_notify_spamadmin = 'spam.police@example.com';
+$mailfrom_notify_admin     = 'virusalert@localhost';
+$mailfrom_notify_recip     = 'virusalert@localhost';
+$mailfrom_notify_spamadmin = 'spam.police@localhost';
 
 # 'From' HEADER FIELD for sender and admin notifications.
 # This should be a replyable address, see rfc1894. Not to be confused
@@ -285,7 +285,7 @@
 
 # $virus_admin = undef;   # do not send virus admin notifications (default)
 # $virus_admin = {'not.example.com' => '', '.' => 'virusalert@example.com'};
-$virus_admin = 'virus-admin@example.com';
+$virus_admin = 'virus-admin@localhost';
 
 # equivalent to $virus_admin, but for spam admin notifications:
 # $spam_admin = undef;    # do not send spam admin notifications (default)
@@ -309,7 +309,7 @@
 #   or a directory (no trailing slash)
 # (the default value is undef, meaning no quarantine)
 #
-$QUARANTINEDIR = '/var/virusmails';
+$QUARANTINEDIR = '/var/spool/amavis/virusmails';
 
 
 # A finer control of quarantining is available through variable
@@ -638,7 +638,7 @@
 # a hash lookup table can be read from a file,
 # one address per line, comments and empty lines are permitted:
 #
-read_hash(\%whitelist_sender, '/var/amavis/whitelist_sender');
+read_hash(\%whitelist_sender, '/var/spool/amavis/whitelist_sender');
 
 # ... or set directly:
 map { $whitelist_sender{lc($_)}=1 } (qw(
Only in amavisd-new-20021227: amavisd~
